libraries/plugins/peerplays_sidechain/sidechain_net_handler_bitcoin.cpp

```diff
@@ -1781,9 +1818,9 @@ std::string sidechain_net_handler_bitcoin::create_transaction_standalone(const s
       tx.outputs().push_back(bout);
    }
 
-   std::string tx_raw = encode_base16(tx.to_data());
+   std::string tx_raw = save_tx_data_to_string(tx.to_data(), in_amounts);
 
```

Comment:
Why not using libbitcoin functions?

Reply:
This function serializes not only transaction itself but input amounts info too that is not part of the transaction information. It is our custom format for "transaction + inputs amounts",  there is no libbitcoin function for this.